### ISSUE TITLE
Bug/intended? 

### ISSUE THREAD

**Issue by freakone20** [SlimeKnights/TinkersConstruct#3569]
there is several problem that i found when using TiCon tool
-Does modifier like haste and sharpness now has maximum level (5 in my case) (bug/intended)? 
-Mining speed of block that doesn't  need tool to break

**Comment by bonii-xx** [comment]
>-Does modifier like haste and sharpness now has maximum level (5 in my case) (bug/intended)?

Yes. With TiC itself you can have at most 4 modifiers on a tool. If you need more than 5 levels, you should probably consider switching to creative mode.

>-Mining speed for block that doesn't need tool to break using pick with Crumbling traits is statis(not affected by hastened modifier) (bug/intended)?

Not sure what exactly you mean, which blocks are affected by that?

**Comment by freakone20** [comment]
sand, dirt, gravel, wood etc. embossing my pickaxe with Knightslime head to get the crumble traits

so sad, for the first one. using tinker tool leveling in my pack, and with that limitations... but its fine

**Comment by bonii-xx** [comment]
Crumbling works pretty well on sand/gravel/etc. Maybe your tools overall speed just is too slow to notice the difference? Or another trait on your tool breaks it, addon materials tend to do that sometimes.

**Comment by freakone20** [comment]
yes there is big difference before and after putting Crumble, but after putting crumble i tried to add lots of redstone , but the mining speed is still the same in my opinion  (for dirt, wood etc)

**Comment by freakone20** [comment]
aah, thanks for the respond

**Comment by bonii-xx** [comment]
From TiCs side everything is working as intended. If you're still noticing any issues it's related to another mod.
